VocaDbWeb/Content/Styles/ExtLinks.less

```diff
@@ -30,7 +30,7 @@ a.extLink {
 		background-image: url("/Content/ExtIcons/atwiki.png");
 	}
 
-	&[href^="http://space.bilibili.tv/"], &[href^="http://www.bilibili.tv/"], &[href^="http://space.bilibili.com/"], &[href^="https://space.bilibili.tv/"], &[href^="https://www.bilibili.tv/"], &[href^="https://space.bilibili.com/"] {
+	&[href*="bilibili.tv/"], &[href*="bilibili.com/"] {

```

Comment:
Maybe href*=".bilibili.tv" (including the dot). Better to make it as specific as possible, if the dot is always there.

Reply:
Yes, there's no such things as "bilibili.com" so far. The dot is always there, you may add it.